### PR TITLE
Feature/image s3 acquisition 

### PR DESCRIPTION
**Pull request by jb-hernandez**
DATAOPS-14164  https://jira.placester.net/browse/DATAOPS-14164

### CODE REVIEW

File: src/app/listingdata/SetListingData.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { exit } from "process"
+import e from 'express'
+import { exit } from 'process'
```
Comment (StArBuLLeT) [inline src/app/listingdata/SetListingData.ts:2]: lets remove this not used

File: src/app/listingdata/SetListingData.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import { exit } from "process"
+import e from 'express'
```
Comment (StArBuLLeT) [inline src/app/listingdata/SetListingData.ts:1]: lets remove this not used

Comment (kfontanilla) [inline src/app/listingdata/SetListingData.ts]: Lets remove this not used

File: src/app/listingdata/SetListingData.ts (hunk 1 above)
Comment (kfontanilla) [inline src/app/listingdata/SetListingData.ts]: lets remove this not used

File: src/server.ts [hunk 3]
```diff
@@ -31,7 +31,7 @@ class Server {
   start() {
     this._configure()
 
-    const port = 8126;
+    const port = process.env.NODE_ENV === 'local' ? 8126 : 8126;
```
Comment (kfontanilla) [inline src/server.ts]: revert this line, will throw error on internal deployment

**Pull request merged**